doc/project/checklist.md

```diff
@@ -295,7 +304,7 @@ Following categories of post-V1 tests focused at in the DV plan review (as appli
 
 ### V2_CHECKLIST_SCOPED
 
-Reviewed V2 checklist to understand scope and estimate effort.
+Review V2 checklist to understand scope and estimate effort.
```

Comment:
To match the rest of the document, this should probably actually be "V2 checklist reviewed".